modules/odc-monitoring-your-project-metrics.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * applications/odc-monitoring-project-and-application-metrics-using-developer-perspective.adoc
+
+[id="odc-monitoring-your-project-metrics_{context}"]
+= Monitoring your project metrics
+
+After you create applications in your project and deploy them, you can use the *Developer* perspective in the web console to see the metrics for your project.
+On the left navigation panel of the *Developer* perspective, click *Monitoring* to see the *Dashboard*, *Metrics*, and *Events* for your project.
```

Comment:
Could this sentence with its "click" instruction instead be the first step of the Procedure possibly?